Build failure

Building leads to failure. Built on Intel MacBook Pro.

>sw_vers
ProductName:		macOS
ProductVersion:		14.7.4
BuildVersion:		23H420

>docker --version
Docker version 28.2.2, build e6534b4

>git log
commit 7dde48e2b54192ae5911ebdff951978f00dd3831 (HEAD -> main, origin/main, origin/HEAD)
Author: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Date:   Wed Jun 25 14:08:32 2025 +0200
...

>docker buildx --builder ksml build --load -t axual/ksml:local --target ksml -f Dockerfile .

``` docker
131.3 [ERROR] Errors:
131.3 [ERROR]   BasicStreamRunTest.testFilterAvroRecords:111 » Parse KSML parse error: Error in input node test: KSML parse error: Error in input node test->streams: Error in stream definition "sensor_source"
131.3 [INFO]
131.3 [ERROR] Tests run: 161, Failures: 0, Errors: 1, Skipped: 0
131.3 [INFO]
131.3 [INFO] ------------------------------------------------------------------------
131.3 [INFO] Reactor Summary for ksml-parent 1.1.0-SNAPSHOT:
131.3 [INFO]
131.3 [INFO] ksml-parent ........................................ SUCCESS [  1.445 s]
131.3 [INFO] KSML Data Library .................................. SUCCESS [ 10.137 s]
131.3 [INFO] KSML Data Library - AVRO ........................... SUCCESS [  1.460 s]
131.3 [INFO] KSML Data Library - BINARY ......................... SUCCESS [  0.654 s]
131.3 [INFO] KSML Data Library - CSV ............................ SUCCESS [  0.870 s]
131.3 [INFO] KSML Data Library - JSON ........................... SUCCESS [  0.697 s]
131.3 [INFO] KSML Data Library - PROTOBUF ....................... SUCCESS [  1.151 s]
131.3 [INFO] KSML Data Library - XML ............................ SUCCESS [  0.929 s]
131.3 [INFO] KSML Data Library - SOAP ........................... SUCCESS [  0.746 s]
131.3 [INFO] KSML ............................................... FAILURE [01:50 min]
131.3 [INFO] KSML Queryable State Store ......................... SKIPPED
131.3 [INFO] KSML Kafka clients ................................. SKIPPED
131.3 [INFO] KSML Runner ........................................ SKIPPED
131.3 [INFO] Jacoco report aggregator ........................... SKIPPED
131.3 [INFO] ------------------------------------------------------------------------
131.3 [INFO] BUILD FAILURE
131.3 [INFO] ------------------------------------------------------------------------

--- a/ksml/src/test/java/io/axual/ksml/BasicStreamRunTest.java
+++ b/ksml/src/test/java/io/axual/ksml/BasicStreamRunTest.java
@@ -68,7 +68,7 @@ class BasicStreamRunTest {
         ExecutionContext.INSTANCE.notationLibrary().register(jsonNotation);
 
         try {
-            final var schemaDirectoryURI = ClassLoader.getSystemResource("pipelines").toURI();
+            final var schemaDirectoryURI = ClassLoader.getSystemResource("schemas").toURI();
             final var schemaDirectory = schemaDirectoryURI.getPath();
             System.out.println("schemaDirectory = " + schemaDirectory);
             ExecutionContext.INSTANCE.schemaLibrary().schemaDirectory(schemaDirectory);

